server/views/pages/bookedtoday/summaryWithRecord.njk

```diff
@@ -18,11 +18,9 @@
         <div class="govuk-body">
             <div class="govuk-grid-row">
                 <div class="govuk-grid-column-one-half">
-                    {% if singleMatch.prisonNumber or singleMatch.pncNumber %}
                         <span class="govuk-caption-l">
-                            {% if singleMatch.prisonNumber %}Prison number: {{ singleMatch.prisonNumber }}{% endif %}{% if singleMatch.prisonNumber and singleMatch.pncNumber %} | {% endif %}{% if singleMatch.pncNumber %}PNC: {{ singleMatch.pncNumber }}{% endif %}
+                            Prison number: {{ singleMatch.prisonNumber }} | PNC: {{ singleMatch.pncNumber }}
```

Comment:
removed conditional as prison number and PNC number always present for single match